src/Migration/Command/GenerateCommand.php

```diff
@@ -89,6 +90,88 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return 1;
         }
 
+        $settings = $this->getGeneratorSettings($input, $environment);
+        
+        $output->writeln('<info>using config file</info> ' . ($settings['config_file'] ?? null));
+        $output->writeln('<info>using migration path</info> ' . ($settings['migration_path'] ?? null));
+        $output->writeln('<info>using schema file</info> ' . ($settings['schema_file'] ?? null));
+
+        $generator = $this->getMigrationGenerator($settings, $input, $output, $environment);
+
+        return $generator->generate();
+    }
+
+    /**
+     * @param string $adapter_name
+     *
+     * @return bool true if adapter with specified name is supported
+     */
+    protected function isAdapterSupported($adapter_name)
```

Comment:
Please use $lowerCamelCase variable names and add type declarations when ever possible. Thanks.